packages/salesforce/src/Adaptor.js

```diff
@@ -148,16 +148,23 @@ export function retrieve(sObject, id, callback) {
  * Execute an SOQL query.
  * Note that in an event of a query error,
  * error logs will be printed but the operation will not throw the error.
+ *
+ * This function makes an API call to retrieve query results.
```

Comment:
Can we simplify to:

>This function is subject to rate limits imposed by the Salesforce API This function will return `2,000` results per request in API versions since 28.0, or 200 results in older versions.

Reply:
Having said that, I'd rather that we say as little as possible (`this function is subject to rate limits by the Salesforce API`), and then link to an SF doc page. But only if there's a good place to link to.